pyjs9/__init__.py

```diff
@@ -225,17 +231,17 @@ def send(self, obj, msg='msg'):
         >>> js9.send({'cmd': 'SetColormap', 'args': ['red']})
         'OK'
         """
-        if obj == None:
+        if obj is None:
             obj = {}
         obj['id'] = self.id
         jstr = json.dumps(obj)
         try:
-            url = urllib.urlopen(self.host + '/' + msg, jstr)
+            url = urlopen(self.host + '/' + msg, jstr.encode('utf8'))
```

Comment:
@ericmandel – Currently this doesn't work ... I get this error in Python 2 and 3:
```
urllib.error.HTTPError: HTTP Error 400: unknown command in POST request: image
```
(In Python 2 it's hidden a bit because of your try .. catch)

I can reproduce the issue like this:
```
import six
six.moves.urllib.request.urlopen('http://localhost:2718/image')
```

This works in Python 2:
```
>>> import urllib
>>> r = urlopen('http://localhost:2718/image')
>>> r.code
400
```

So apparently `six.moves.urllib.request.urlopen` calls `urllib2.urlopen` on Python 2, which raises a `HTTPError` for a 400 return code, whereas `urllib.urlopen` didn't.

I'm not sure what to do here because I'm not familiar with the code. Catch and ignore the HTTPError? Call some other test URL?